### ISSUE TITLE
Long (duration) files getting shortened after conversion

### ISSUE THREAD

**Issue by rbqbjyur** [enzo1982/freac#162]
**Describe the bug**
I have only been converting long ".mp3"s and ".m4b"s to a single ".opus", unsure if/what other formats are affected.

When converting very long files (audiobooks in particular), which are 3+ (sometimes as long as 30) hours long, they come out a 
lot shorter.

**Details**
As an example here is a 32.5 hour long .mp3 details pre going through fre:ac
```
Format                                   : MPEG Audio
Format version                           : Version 2
Format profile                           : Layer 3
Format settings                          : Joint stereo / MS Stereo
Duration                                 : 32 h 26 min
Bit rate mode                            : Variable
Bit rate                                 : 64.0 kb/s
Channel(s)                               : 2 channels
Sampling rate                            : 22.05 kHz
Frame rate                               : 38.281 FPS (576 SPF)
Writing library                          : LAME3.100
```

I then convert to opus with following settings;
```
Encoding mode:	Voice
Bandwidth:	Fullband
Bitrate:	48 kbps
Frame Length:	20ms
```

The output .opus file has the below;
```
Format                                   : Opus
Duration                                 : 7 h 34 min
Channel(s)                               : 2 channels
Channel layout                           : L R
Sampling rate                            : 24.0 kHz
Writing library                          : libopus 1.3.1
```

**System**
 - OS: Fedora
 - CPU: 3900X
 - RAM: 64 GB

**Additional context**
Strictly speaking, I don't know if this is also happening with shorter files, and perhaps only noticeable when using very long files. Will try to look into what else is affected.

**Comment by rbqbjyur** [comment]
Just noticed you responded in #150 , many thanks for all your work.

**Comment by enzo1982** [comment]
Fixed now via enzo1982/BoCA@4fb9fa91.

The current continuous AppImage builds already have the fix.

**Comment by rbqbjyur** [comment]
Thanks, can confirm (using continuous build 2020-10-22) converting the same .mp3 now provides the below;

```
Format                                   : Opus
Duration                                 : 32 h 26 min
Channel(s)                               : 2 channels
Channel layout                           : L R
Sampling rate                            : 24.0 kHz
Writing library                          : libopus 1.3.1
```

Tested with several other files in the 10s of hours, all came out perfect.
Thanks again.
